src/runmode-dpdk.c

```diff
@@ -1191,6 +1208,19 @@ static void PortConfSetChsumOffload(const DPDKIfaceConfig *iconf,
     }
 }
 
+static void PortConfSetVlanOffload(const DPDKIfaceConfig *iconf,
+        const struct rte_eth_dev_info *dev_info, struct rte_eth_conf *port_conf)
+{
+    if (iconf->vlan_strip_enabled) {
+        SCLogConfig("%s: hardware vlan stripping enabled", iconf->iface);
+        if (dev_info->rx_offload_capa & RTE_ETH_RX_OFFLOAD_VLAN_STRIP) {
+            port_conf->rxmode.offloads |= RTE_ETH_RX_OFFLOAD_VLAN_STRIP;
+        } else {
+            SCLogWarning("%s: hardware vlan stripping not supported", iconf->iface);
```

Comment:
The warning could be reworded to something like 
%s: hardware VLAN stripping enabled but not supported, disabling